src/spacy/config.clj

```diff
@@ -0,0 +1,10 @@
+(ns spacy.config
+  (:require [aero.core :as aero]
+            [clojure.java.io :as io]))
+
+(defn config [profile]
+  {:pre [(contains? #{:dev :prod} profile)]}
+  (aero/read-config (io/resource "config.edn") {:profile profile}))
```

Comment:
In the future (when we have two separate configuration options), will we read from two separate files for `dev` and `prod`?

Reply:
We can either have two separate files or use the following approach:

```clojure
{:webserver
  {:port #profile {:default 8000
                   :dev 8001
                   :test 8002}}}
```

See also https://github.com/juxt/aero#profile

Reply:
Ah... interesting. Ok. Looks good 😄